.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  env: {
+    browser: false,
+    commonjs: true,
+    es6: true,
+  },
+  extends: ["eslint:recommended", "plugin:node/recommended"],
+  globals: {
+    Atomics: "readonly",
```

Comment:
Out of curiosity: where do these come from?

Reply:
To be honest, it came from the `eslint --init` command I think.